daemon/oci_linux.go

```diff
@@ -463,11 +463,9 @@ func setMounts(daemon *Daemon, s *specs.Spec, c *container.Container, mounts []c
 		}
 
 		if m.Source == "tmpfs" {
-			opt := []string{"noexec", "nosuid", "nodev", volume.DefaultPropagationMode}
+			opt := []string{"noexec", "nosuid", "nodev", volume.DefaultPropagationMode, "size=65536k"}
```

Comment:
The "usual" part bothers me a bit, it sounds like something that *may* be different between distros. So perhaps we should explicitly merge user options with the defaults?